repository/tests/integration_test.rs

```diff
@@ -0,0 +1,111 @@
+use simperby_common::*;
+use simperby_network::{NetworkConfig, Peer};
+use simperby_repository::{raw::*, *};
+use simperby_test_suite::*;
+
+#[ignore]
+#[tokio::test]
+async fn basic_1() {
+    let port = 8801;
+    let n = 4;
+    let (rs, keys) = generate_standard_genesis(n);
+    let config = Config {
+        mirrors: Vec::new(),
+        long_range_attack_distance: 1,
+    };
+    let network_config = NetworkConfig {
+        network_id: "test".to_string(),
+        port: None,
+        members: vec![keys[0].0.clone(), keys[1].0.clone()],
+        public_key: keys[1].0.clone(),
+        private_key: keys[1].1.clone(),
+    };
+    let peers = vec![Peer {
+        public_key: keys[0].0.clone(),
+        name: "server-node".to_owned(),
+        address: format!("127.0.0.1:{}", 1).parse().unwrap(),
+        ports: vec![("repository".to_owned(), port)].into_iter().collect(),
+        message: "".to_owned(),
+        recently_seen_timestamp: 0,
+    }];
+
+    let server_node_dir = create_temp_dir();
+    setup_pre_genesis_repository(&server_node_dir, rs.clone()).await;
+    let mut server_node_repo = DistributedRepository::new(
+        RawRepositoryImpl::open(&server_node_dir).await.unwrap(),
+        config.clone(),
+    )
+    .await
+    .unwrap();
+    server_node_repo.genesis().await.unwrap();
+
+    tokio::spawn(async move { server::run_server(&server_node_dir, 7301).await });
```

Comment:
7301 is default port for what?

Reply:
It's not something default. It's just an arbitrary number for the test.